### ISSUE TITLE
Please provide  MySQL Database Backup file in source code

### ISSUE THREAD

**Issue by TariqJavaid** [waqasraza123/simple-attendance-system-in-php--mysql--bootstrap-#1]
Salam Waqar,

If possible please help to add-in MySQL database backup file also inside the source code so we can easily restore use the system. 

Thank you,

Br,
Tariq

**Comment by waqasraza123** [comment]
@TariqJavaid you can create the database as per your requirements and modify the code accordingly, I am not managing this anymore.

**Comment by michael-milette** [comment]
Hi @TariqJavaid,

I tried this out today just for kicks (I was curious). It seems to work if you make a few changes. In pull request #3, I've included 3 fixes to the PHP files and some documentation for anyone who might want to get it running. I does not include improvements to the intended functionality.

This is obviously not a finished application. However @waqasraza123 is providing a start if you want to develop your own Attendance web application.

Best regards,

Michael Milette
